docs/2.deploy/20.providers/sevalla.md

```diff
@@ -0,0 +1,44 @@
+# Sevalla
+
+> Deploy Nitro apps to Sevalla.
+
+**Preset:** `sevalla`
+
+:read-more{title="sevalla.com" to="https://sevalla.com"}
+
+## Setup
+
+1. [Create a new app](https://docs.sevalla.com/applications/get-started/add-an-application) on Sevalla and connect it to your Git repository.
+2. Set the **application name**, **region**, and **pod size** (you can start with 0.5 CPU / 1GB RAM).  
+3. Click **Create**, but skip the deploy step for now.  
+4. In your Sevalla application dashboard, click **Environment variables** on the sidebar and set `NITRO_PRESET` to `sevalla`.
```

Comment:
Could svella CI somehow set this env? (or if it has a known env variable, we can add it to [std-env](https://github.com/unjs/std-env/blob/main/src/providers.ts) for zero config